src/main/java/emt/init/EMTRecipes.java

```diff
@@ -1091,54 +1092,58 @@ public static void registerLateRecipes() {
 
         /** IC2 Stuff related recipes **/
 
-        /* Ore Clusters Macerator Recipes */
-        ItemStack ironClusterRecipe = IC2Items.getItem("smallIronDust").copy();
-        ironClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 16)),
-                null,
-                true,
-                ironClusterRecipe);
-
-        ItemStack goldClusterRecipe = IC2Items.getItem("smallGoldDust").copy();
-        goldClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 31)),
-                null,
-                true,
-                goldClusterRecipe);
-
-        ItemStack copperClusterRecipe = IC2Items.getItem("smallCopperDust").copy();
-        copperClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 17)),
-                null,
-                true,
-                copperClusterRecipe);
-
-        ItemStack tinClusterRecipe = IC2Items.getItem("smallTinDust").copy();
-        tinClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 18)),
-                null,
-                true,
-                tinClusterRecipe);
-
-        ItemStack silverClusterRecipe = IC2Items.getItem("smallSilverDust").copy();
-        silverClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 19)),
-                null,
-                true,
-                silverClusterRecipe);
-
-        ItemStack leadClusterRecipe = IC2Items.getItem("smallLeadDust").copy();
-        leadClusterRecipe.stackSize = 22;
-        ((BasicMachineRecipeManager) Recipes.macerator).addRecipe(
-                new RecipeInputItemStack(new ItemStack(ConfigItems.itemNugget, 1, 20)),
-                null,
-                true,
-                leadClusterRecipe);
+        final boolean isDreamcraftLoaded = Loader.isModLoaded("dreamcraft");
+
+        if (isDreamcraftLoaded) {
```

Comment:
If idea is to nuke IC2 recipes then these conditions should be inverted, shouldn't it? I. e. IC2 recipes should be registered only without NHCore. Or am I misunderstood something?

Reply:
that's right lmao